lib/Controller/PersonController.php

```diff
@@ -56,9 +62,18 @@ public function index() {
 		$persons = $this->personMapper->findAll($this->userId);
 		foreach ($persons as $person) {
 			$cluster = [];
+			$faces = [];
+			$personFaces = $this->faceNewMapper->findFacesFromPerson($this->userId, $person->getId(), $model);
+			foreach ($personFaces as $personFace) {
+				$image = $this->imageMapper->find($this->userId, $personFace->getImage());
+				$face = [];
+				$face['id'] = $personFace->getId();
+				$face['file-id'] = $image->getFile();
+				$faces[] = $face;
+			}
```

Comment:
It is not ideal so many embedded queries, but even adding more data, reduce the final query from 111Kb to 33Kb. :sweat_smile: